### PR TITLE
Ref lvjavs 967 – refactoring user controller, add user-calendar.jsp

### PR DESCRIPTION
**Pull request by Yevhensh**
i can also squash all my commits by merging. So in master u'll se only one commit

### CODE REVIEW

File: src/main/java/ua/softserveinc/tc/constants/UserConstants.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 
 public final class UserConstants {
+  public static final String USER_CALENDAR = "user-calendar";
+
```
Comment (Tat0) [inline src/main/java/ua/softserveinc/tc/constants/UserConstants.java]: please add another constant to the views name.

Comment (Yevhensh) [inline src/main/java/ua/softserveinc/tc/constants/UserConstants.java]: this constant is common to
public static final String MANAGER_CALENDAR = "manager-calendar";

Comment (Tat0) [inline src/main/java/ua/softserveinc/tc/constants/UserConstants.java]: Take a look on the class of mine . 

There is two type of constans. One is our URL adress and another is our view name.

Comment (Yevhensh) [inline src/main/java/ua/softserveinc/tc/constants/UserConstants.java]: okay

**Pull request merged**